qa/tasks/cephadm.py

```diff
@@ -391,7 +391,7 @@ def ceph_bootstrap(ctx, config):
         ssh_pub_key = teuthology.get_file(
             remote=bootstrap_remote,
             path='{}/{}.pub'.format(testdir, cluster_name)
-        ).strip()
+        ).decode('ascii').strip()
```

Comment:
Not sure I understand why we need to decode this? From my understanding the keys should be always binary.

Reply:
Ah... it is literally cat it later...